server/core_authenticate.go

```diff
@@ -477,6 +479,18 @@ func AuthenticateGoogle(ctx context.Context, logger *zap.Logger, db *sql.DB, cli
 			return "", "", false, status.Error(codes.Unauthenticated, "Error finding or creating user account.")
 		}
 
+		// Check if display name / avatar received from Google different from those in the DB
+		if dbDisplayName.String != googleProfile.Name || dbAvatarUrl.String != googleProfile.Picture {
+			logger.Info("About to check Google Id's details now. Found differences")
```

Comment:
I think this log line should be removed.

Reply:
Yes! Removed in the next commit.